packages/fxa-event-broker/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "fxa-event-broker",
+  "version": "1.137.1",
+  "description": "Firefox Accounts Event Broker",
+  "scripts": {
+    "build": "./node_modules/typescript/bin/tsc",
+    "tslint": "./node_modules/tslint/bin/tslint -p .",
+    "test": "./node_modules/mocha/bin/mocha -r ts-node/register test/**/*.spec.ts",
+    "start-processor": "node ./dist/bin/worker.js"
```

Comment:
can we just add a `start` script here that runs a `build` and calls start-processor?

Reply:
Sure.